src/software/simulated_tests/validation_functions/robot_received_ball_validation.cpp

```diff
@@ -0,0 +1,25 @@
+#include "software/simulated_tests/validation_functions/robot_received_ball_validation.h"
+
+#include "software/ai/evaluation/robot.h"
+#include "software/logger/logger.h"
+
+
+void robotReceivedBall(RobotId robot_id, std::shared_ptr<World> world_ptr,
+                       ValidationCoroutine::push_type& yield)
+{
+    auto ball_near_dribbler = [robot_id](std::shared_ptr<World> world_ptr) {
+        std::optional<Robot> robot = world_ptr->friendlyTeam().getRobotById(robot_id);
+        if (!robot.has_value())
+        {
+            LOG(FATAL);
```

Comment:
Something descriptive should be piped to this log. 
`LOG(FATAL) << "oh no it's broken :( ;`